__tests__/app.test.js

```diff
@@ -12,38 +12,76 @@ afterAll(() => {
   return db.end();
 });
 
-describe("/api/categories", () => {
-  it("GET: 200, should return an array of the categories", () => {
+describe("app", () => {
+  it("GET: 404, handle incorrect endpoint", () => {
     return request(app)
-      .get("/api/categories")
-      .expect(200)
+      .get("/not_a_real_endpoint")
+      .expect(404)
       .then(({ body }) => {
-        expect(body).toEqual({
-          categories: [
-            {
-              slug: "euro game",
-              description: "Abstact games that involve little luck",
-            },
-            {
-              slug: "social deduction",
-              description:
-                "Players attempt to uncover each other's hidden role",
-            },
-            {
-              slug: "dexterity",
-              description: "Games involving physical skill",
-            },
-            {
-              slug: "children's games",
-              description: "Games suitable for children",
-            },
-          ],
-        });
+        expect(body).toEqual({ msg: "Path not found!" });
       });
   });
-  it("GET: 404, handle incorrect endpoint", () => {
-    return request(app).get("/api/categorie").expect(404);
+  describe("/api/categories", () => {
+    it("GET: 200, should return an array of the categories", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toEqual({
```

Comment:
We could test this in a firmer way by testing the keys of the objects rather than any set values - data may change and then the test fails